cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -619,4 +601,4 @@ def _deserialize_circuit_op(
         )
 
 
-CIRCUIT_SERIALIZER = CircuitSerializer('v2_5')
+CIRCUIT_SERIALIZER = CircuitSerializer()
```

Comment:
Will this barf server-side if the language is not present?  I can't remember how we determine whether this is v2 or v2.5 protos currently.

Reply:
Current it does. My plan is to update this logic on server side as well. @maffoo and I confirmed this should be the order of merging:

1. Remove SerializableGateSet
2. Change server side to only accept CircuitSerializer, and throw an error if the `gate_set` field doesn't match CircuitSerializer
3. Update CircuitSerializer on Cirq side

We also agreed to not remove `Program.language.gate_set` in the end, so that users on old Cirq versions won't fail silently when using QCS. If we choose to remove it in the future, it would be safer to do it later after more users upgrade past Cirq 1.0.

Reply:
Yeah, I think #2 needs to happen first, since server side needs to support the new client.  (maybe throw an error if gate_set is not empty and does not equal v2_5 for now).